osbs/cli/main.py

```diff
@@ -704,16 +704,16 @@ def cli():
                               help="perform a scratch build")
     build_parser.add_argument("--yum-proxy", action='store', required=False,
                               help="set yum proxy to repos from koji/add-yum-repo params")
+    build_parser.add_argument('--release', action='store', required=False,
+                              help='release value to use')
+    build_parser.add_argument('--arrangement-version', action='store', required=False,
+                              help='version of inner template to use')
```

Comment:
The group parser is only used for grouping the help text. Although it looked like it, we actually never enforced that `arrangement-version` was not given to a prod build.

We can either move `arrangement-version` back to where it was. Or add the restriction to `create_prod_build` (which btw actually lists `arrangement_version` as valid parameter).